api/campusdish_interface.py

```diff
@@ -14,33 +15,47 @@ def get_menu_data(location, meal_id, date):
     response = requests.get(
         f'https://uci.campusdish.com/api/menu/GetMenus?locationId={location_id}&periodId={MEAL_TO_PERIOD[meal_id][0]}&date={date}'
     )
-    if response.status_code==200:
+    if response.status_code == 200:
         payload = response.json()
         if 'Menu' in payload:
             return payload['Menu']
         else:
-            raise KeyError(f'Key "Menu" not found in campusdish response object. Response payload below:\n{payload}')
+            raise KeyError(
+                f'Key "Menu" not found in campusdish response object. Response payload below:\n{payload}')
     else:
         print("Response error message: ", response.json())
         response.raise_for_status()
 
-def get_schedule_data(location, date):
+
+def get_schedule_data(restaurant: str) -> dict:
     '''
-    Given a valid location and date,
-    perform get request for the schedule_json
+    Given the restaurant name,
+    perform get request, then parse the HTML code using BeautifulSoup 4
+    return a dictionary
     '''
-    response = requests.get(
-        f'https://uci.campusdish.com/api/menu/GetMenuPeriods?locationId={location}&date={date}'
-    )
-    if response.status_code==200:
-        return response.json()['Result']
+    url = 'https://uci.campusdish.com/LocationsAndMenus/'
+    if restaurant == 'Anteatery':
+        url += 'TheAnteatery'
     else:
-        print("Response error message: ", response.json())
-        response.raise_for_status()
+        url += restaurant
+    schedule = {}
+    soup = bs(requests.get(url).text, 'html.parser')
+    meal_period = soup.select('.mealPeriod')[:5]
+    location_times = soup.select('span[class=location__times]')[:5]
+    for idx, meal in enumerate(meal_period):
+        meal = meal.getText().lower()
+        times = location_times[idx].getText().split(' - ')
+        # start = int(times[0][0:-5])
```

Comment:
> I fixed the time in the dictionary

Can you make them integers again? The frontend iOS code expects to be able to parse them that way:
![image](https://user-images.githubusercontent.com/48658337/206988685-d6611b2b-7a68-456e-a070-c39cccf94385.png)

Although you are right that it's not immediately clear what the integers mean, so we should document it somewhere. As a start, just add a docstring comment to the schedule function explaining how to interpret the output.